cluster/store_test.go

```diff
@@ -35,12 +35,12 @@ func (suite *EtcdDependentSuite) TestKVGet() {
 
 	tmpKV := clientv3.NewKV(c)
 	expected := "uwu1"
-	_, err = tmpKV.Put(ctx, "store/raccoon1", expected)
+	_, err = tmpKV.Put(ctx, "store/raccoon1/", expected)
 	require.NoError(t, err)
-	_, err = tmpKV.Put(ctx, "store/raccoon2", "uwu2")
+	_, err = tmpKV.Put(ctx, "store/raccoon2/", "uwu2")
```

Comment:
The test would still pass without these additons right?

Reply:
The change on line 38 is necessary because a "/" is appended to the key when using store.Get now.